routes/api.php

```diff
@@ -93,9 +93,20 @@
 Route::post('/Sharing::add', [Administration\SharingController::class, 'add']);
 Route::post('/Sharing::delete', [Administration\SharingController::class, 'delete']);
 
-Route::post('/WebAuthn::register/gen', [Administration\WebAuthController::class, 'generateRegistration']);
-Route::post('/WebAuthn::register', [Administration\WebAuthController::class, 'verifyRegistration']);
-Route::post('/WebAuthn::login/gen', [Administration\WebAuthController::class, 'generateAuthentication']);
-Route::post('/WebAuthn::login', [Administration\WebAuthController::class, 'verifyAuthentication']);
-Route::post('/WebAuthn::list', [Administration\WebAuthController::class, 'list'])->middleware(['admin']);
-Route::post('/WebAuthn::delete', [Administration\WebAuthController::class, 'delete'])->middleware(['admin']);
+// Route::post('/WebAuthn::register/gen', [Administration\WebAuthController::class, 'generateRegistration']);
+// Route::post('/WebAuthn::register', [Administration\WebAuthController::class, 'verifyRegistration']);
+// Route::post('/WebAuthn::login/gen', [Administration\WebAuthController::class, 'generateAuthentication']);
+// Route::post('/WebAuthn::login', [Administration\WebAuthController::class, 'verifyAuthentication']);
+Route::post('/WebAuthn::list', [WebAuthn\WebAuthnManageController::class, 'list']);
+Route::post('/WebAuthn::delete', [WebAuthn\WebAuthnManageController::class, 'delete']);
+
+// WebAuthn Routes
+Route::post('/WebAuthn::register/options', [\App\Http\Controllers\WebAuthn\WebAuthnRegisterController::class, 'options'])
+	->name('webauthn.register.options');
```

Comment:
Why is the comment placed between the 2nd and the 3rd route and not at the top, i.e. before the 1st route? I am unsure whether we need that comment at all, but if we put this comment here it should be placed at the top like a caption.

```suggestion
// WebAuthn Routes
Route::post('/WebAuthn::list', [WebAuthn\WebAuthnManageController::class, 'list']);
Route::post('/WebAuthn::delete', [WebAuthn\WebAuthnManageController::class, 'delete']);
Route::post('/WebAuthn::register/options', [\App\Http\Controllers\WebAuthn\WebAuthnRegisterController::class, 'options'])
	->name('webauthn.register.options');
```

Reply:
Fixed in [4cdfc8c](https://github.com/LycheeOrg/Lychee/pull/1469/commits/4cdfc8cfffc94a3b571ec8c68b2fa6063a0e5d1f)